amm/runtime/src/main/scala/ammonite/runtime/ImportHook.scala

```diff
@@ -39,6 +39,8 @@ object ImportHook{
     Seq("url") -> URL,
     Seq("file") -> File,
     Seq("exec") -> Exec,
+    Seq("repo", "maven") -> MavenRepo,
```

Comment:
I would have gone for just `$repo`, like
```scala
import $repo.`https://jcenter.bintray.com`
import $repo.`ivy:…`
```

In a first time, you could go for Maven or Ivy by checking if the repository string starts with `ivy:`.

coursier has [a method](https://github.com/coursier/coursier/blob/3f310cad337ff44cae1490618a717904b3b9a429/modules/coursier/jvm/src/main/scala/coursier/internal/PlatformRepositoryParser.scala#L14) (accessible via `coursier.parse.RepositoryParser.repository`) to parse both Maven and Ivy repositories at the same time. It needs to be made accessible in [coursier/interface](https://github.com/coursier/interface) so that it could later be used here too.

Reply:
Happy to change it :+1:

Reply:
Changed in https://github.com/lihaoyi/Ammonite/pull/1090/commits/40b0256bf7d4f193b482a751ff6f4b208ae7fa93